python_scripts/functions/data_validation.py

```diff
@@ -220,13 +226,32 @@ def read_and_write_cleaned_data(
         logger.info(f"{name} data for {start_date} written to s3.")
 
 
+def refresh_new_partition(database_name: str, table_name: str, scrape_date: str):
+    athena = boto3.client("athena", region_name=region_name)
+    query_string = f"""alter table awsdatacatalog.{database_name}.{table_name} 
+                add partition (scrape_date = '{scrape_date}')"""
+    logger.info(f"Athena Query: adding {scrape_date} partition to \
+                {database_name}.{table_name}")
+    resp = athena.start_query_execution(QueryString=query_string)
```

Comment:
I think you need to wait for the query to finish to check it executed successfully. You can probably swap out the boto3 calls for `wr.athena.start_query_execution` and then run `wr.athena.wait_query` based on the returned execution id, from wrangler (see [here](https://aws-sdk-pandas.readthedocs.io/en/stable/api.html#amazon-athena)).

Reply:
I knew it was lazy just returning the id, it took me an entire day to figure out that the lack of return value was the cause of the problem in Airflow and i thought it was better to submit some type of progress than waste more time.

I also couldn't understand why the `read_and_write_cleaned_data` function, which also had no return value, didn't cause a problem (TypeError: 'NoneType' object is not callable) when the function was assigned to a variable in main?

That nice that awswrangler raises an error on failed.